docs/3_2_model_exchange_api.adoc

```diff
@@ -211,27 +210,31 @@ with the following meanings:
  In this case, the FMU must determine itself if this event type happens.
 --
 +
-The arguments of the function have the following meaning:
+For Model Exchange, the arguments of the function have the following meaning _[For Co-Simulation, the semantics of the arguments are <<fmi3EnterEventModeCS,different>>.]_:
 +
 --
-* Model Exchange only: `stepEvent` signals a <<step-event>>.
+* `stepEvent` signals a <<step-event>>.
 `stepEvent` must be `fmi3EventTrue` if <<fmi3CompletedIntegratorStep>> returned with `enterEventMode = fmi3True`.
 
-* Model Exchange only: `stateEvent` signals a <<state-event>>, triggered by a zero-crossing of an event indicator.
+* `stateEvent` signals a <<state-event>>, `stateEvent` is triggered by a domain change of an event indicator.
```

Comment:
"domain change" of an event indicator (used in several places)  is not clearly defined and does not seem to be broadly used in literature. 
If we stick with this, we must define it.
Do you mean a change of sign?